tests/unit/CorrectNameRequest.spec.ts

```diff
@@ -396,7 +396,7 @@ describe('CorrectNameRequest', () => {
             }],
             nrNum: 'NR 1234567',
             requestTypeCd: 'GP',
-            request_action_cd: 'CNV',
+            request_action_cd: 'CHG',
             entity_type_cd: 'GP',
```

Comment:
I'm thinking lines 398 and 400 should be SP.

I'm also wondering whether the mixin should verify that the type code matches the current filing type...? Or can a GP NR be used for a SP correction? Please check with PO/BA and report back. Thanks.

Reply:
^^ outstanding question to answer

Reply:
GP NR cannot be used for a SP correction. They must be the same type: GP NR for GP correction and SP NR for SP correction.

Reply:
OK. Can you implement a type check in the NR mixin, or create a new ticket?

Reply:
I'll implement the type check in this PR.

Reply:
After I implemented the type check, then I found out that there is already an entity type checker implemented: [here](https://github.com/bcgov/business-edit-ui/blob/5457c482ab6d6fbee20060f14525f29824f7b224/src/components/common/YourCompany/CompanyName/CorrectNameRequest.vue#L159-L176). If you think this is fine, I will revert my code and then push it.

Reply:
Oh, oops.

Yes, I think the original type check is fine.